### ISSUE TITLE
Wazuh agent v3.10.0 can't be built in CentOS5

### ISSUE THREAD

**Issue by sergiovp** [wazuh/wazuh#3823]
|Wazuh version|Platform|
|---|---|
| 3.10 | CentOS 5|

While building Wazuh 3.10 branch we get the next error on CentOS5:

~~~
wazuh_modules/wm_database.c: In function 'wm_inotify_setup':
      wazuh_modules/wm_database.c:1335: warning: implicit declaration of function 'inotify_init1'
      wazuh_modules/wm_database.c:1335: error: 'IN_CLOEXEC' undeclared (first use in this function)
      wazuh_modules/wm_database.c:1335: error: (Each undeclared identifier is reported only once
      wazuh_modules/wm_database.c:1335: error: for each function it appears in.)
from wazuh_modules/wm_docker.c:14:
      wazuh_modules/wm_control.h:31:7: warning: no newline at end of file
      cc: --start-group: linker input file unused because linking not done
      In file included from wazuh_modules/wmodules.h:94,
                       from wazuh_modules/wm_download.c:12:
      wazuh_modules/wm_control.h:31:7: warning: no newline at end of file
      cc: --start-group: linker input file unused because linking not done
      In file included from wazuh_modules/wmodules.h:94,
                       from wazuh_modules/wm_exec.c:12:
      wazuh_modules/wm_control.h:31:7: warning: no newline at end of file
      cc: --start-group: linker input file unused because linking not done
      make[1]: Leaving directory `/build_wazuh/rpmbuild/BUILD/wazuh-agent-3.10.0/src'
      make: *** [agent] Error 2
      error: Bad exit status from /var/tmp/rpm-tmp.16259 (%build)
~~~

This error is due CentOS5 hasn't got **O_CLOEXEC** flag.
We have to avoid CentOS 5 to execute this flag.

**Comment by vikman90** [comment]
This issue is a duplicate of #3820.
